apps/promisetracker/eslint.config.js

```diff
@@ -0,0 +1,14 @@
+const eslintConfig = require("eslint-config-commons-ui/next");
+
+module.exports = [
+  {
+    settings: {
+      "import/resolver": {
+        webpack: {
+          config: "./eslint.webpack.config.js",
+        },
+      },
+    },
+  },
+  ...eslintConfig,
+];
```

Comment:
since this is almost identical to what we have on [PesaYetu](https://github.com/CodeForAfrica/ui/blob/d9bbe3f518a102b384b6859e163a1a6f19c64cd4/apps/pesayetu/eslint.config.js) I was just wondering if there was any specific reason of not starting with  spreading eslintConfig and then keep any custom configs